bin/install.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+set -eu
+
+THIS_DIR=$(
+  cd $(dirname $0)
+  pwd
+)
+
+USER_REPO_ROOT=$HOME/go/src/github.com/ntsk
```

Comment:
This follows `$GOPATH`. If you want to change another location, you can change this.